src/main/java/reposense/parser/CsvParser.java

```diff
@@ -41,15 +44,17 @@
      *
      * @throws IOException if user-supplied csv file does not exists or is not readable.
      */
-    public static List<RepoConfiguration> parse(Path configFilePath) throws IOException {
-        assert (configFilePath != null);
+    public static List<RepoConfiguration> parse(Path configFolderPath) throws IOException {
```

Comment:
Feels weird that a `CsvParser` is passed a folder to parse, instead of the `csv` file directly.
Perhaps get the `RepoSense` main class to resolve and pass the `csv` file path here?